src/components/LeaderboardTable.vue

```diff
@@ -0,0 +1,51 @@
+<script setup>
+import { defineProps } from "vue";
+const { contribution } = defineProps({
+  contributions: {
+    userName: String,
+    toolsEdited: Number
+  },
+  title: String,
+  profileBaseUrl: String
+})
+</script>
+
+<template>
+  <v-table class="table elevation-2">
+    <template v-slot:default>
+      <thead>
+        <tr>
+          <th class="text-left text-h5 mx-1 my-3" colspan="2">
+            {{ title }}
+          </th>
+        </tr>
+        <tr>
+          <th class="text-left text-subtitle-2 font-weight-bold">
+            User's Name
+          </th>
+          <th class="text-center text-subtitle-2 font-weight-bold">
+            Number of Tools Edited
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr
+          v-for="contribution in contributions"
+          :key="contribution.userName"
+        >
+          <td class="text-left">
+            <a
+              :href="`${profileBaseUrl}${contribution.userName
+                .split(' ')
+                .join('_')}`"
+              target="_blank"
+              >{{ contribution.userName }}</a
+            >
+          </td>
+
```

Comment:
If you wanted to really drill down into components, `/src/components/table_data/UserProfile.vue` is a component that will generate the <td> element with the username and profile link.

You can add `import UserProfile from "./table_data/UserProfile.vue";` to line 3 and replace lines 36-44 with `<UserProfile :user="contribution.userName" />`